Gemfile.lock

```diff
@@ -47,6 +47,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  arm64-darwin-22
```

Comment:
Bundler auto-generated this on my M1 Pro. Checking it in so that we don't see this diff again in future from other M1 users.